### PR TITLE
Run isort on travis

### PR DESCRIPTION
**Pull request by michael-k**
Ping me right before merging it, then I'll run isort to fix all the imports or run it yourself: `isort --recursive --apply`.  Merging this last should lead to the fewest merge conflicts.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 cache: pip
 
 python:
   - "2.7"
 
-install:
-  - pip install -r requirements-dev.txt
-
-script: "py.test -c pytest_ci.ini"
+matrix:
```
Comment (artcz) [inline .travis.yml:11]: what's matrix?

**Comment by matrixise** [comment]
Hi @michael-k 

Why do you need to execute `isort` on Travis?
Usually, we use it during the development.

**Comment by michael-k** [comment]
Running it on travis ensures that it was executed on dev.  Tests fail if someone committed code without incorrect sorted imports.

File: .travis.yml (hunk 1 above)
Comment (michael-k) [reply .travis.yml:11]: https://docs.travis-ci.com/user/customizing-the-build/#build-matrix

It allows you to run multiple jobs in parallel.

See https://travis-ci.org/EuroPython/epcon/builds/409527705 (tests for this PR) vs https://travis-ci.org/EuroPython/epcon/builds/409527705 (tests for default branch)

**Pull request closed without merging**